transformer_engine/common/include/transformer_engine/fused_attn.h

```diff
@@ -0,0 +1,188 @@
+/*************************************************************************
+ * Copyright (c) 2022-2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+ *
+ * See LICENSE for license information.
+ ************************************************************************/
+
+#ifndef TRANSFORMER_ENGINE_FUSED_ATTN_FP8_H_
+#define TRANSFORMER_ENGINE_FUSED_ATTN_FP8_H_
+
+#include "transformer_engine.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/*! \brief Compute dot product attention with packed QKV input.
+ *
+ * Computes: BMM1 -> ScaleMaskSoftmax -> Dropout -> BMM2
```

Comment:
Where is bias in this computation? I think it would be better to explain this computation as a peudocode.

Reply:
That way you could also introduce S tensor.

Reply:
How about this:
`/*! \brief Compute dot product attention with packed QKV input.
 *
 * Computes:
 *  - P = Q * K.T + B
 *  - S = ScaleMaskSoftmax(P)
 *  - D = Dropout(S)
 *  - O = D * V.T`

Reply:
B -> bias (since that is the name you use in the parameters)

Other than that, yeah, I think it works.

Reply:
Done.